### PR TITLE
FIX: PEP8 in utils

### PR DESCRIPTION
**Pull request by manu-tej**
issue #889 is fixed

### CODE REVIEW

File: dipy/utils/_importlib.py [hunk 1]
```diff
@@ -12,8 +13,8 @@ def _resolve_name(name, package, level):
         try:
             dot = package.rindex('.', 0, dot)
         except ValueError:
-            raise ValueError("attempted relative import beyond top-level "
-                              "package")
+            raise ValueError("attempted relative import beyond top-level " +
+                             "package")
```
Comment (manu-tej) [inline dipy/utils/_importlib.py]: 1

**Comment by manu-tej** [comment]
@arokem Completed making different PRs for different files.

File: dipy/utils/optpkg.py [hunk 2]
```diff
@@ -62,7 +63,8 @@ def optional_package(name, trip_msg=None):
     >>> pkg.some_function() #doctest: +IGNORE_EXCEPTION_DETAIL
     Traceback (most recent call last):
         ...
-    TripWireError: We need package not_a_package for these functions, but ``import not_a_package`` raised an ImportError
+    TripWireError: We need package not_a_package for these functions, but
+    ``import not_a_package`` raised an ImportError
```
Comment (arokem) [inline dipy/utils/optpkg.py]: This should still be on one line. It's the expected output in the Python process.

**Comment by arokem** [comment]
Just one small comment. Other than that - looks good!

**Pull request merged**